### PR TITLE
Removed Digitrust From Prebid Server

### PR DESCRIPTION
**Pull request by AlexBVolcy**
Implements https://github.com/prebid/prebid-server/issues/1464

### CODE REVIEW

**Review by bretg — DISMISSED** [review]
Signing off on the rubicon adapter change

File: endpoints/openrtb2/auction.go [hunk 1]
```diff
@@ -1127,10 +1127,6 @@ func (deps *endpointDeps) validateUser(user *openrtb2.User, aliases map[string]s
 		// Creating ExtUser object to check if DigiTrust is valid
```
Comment (mansinahar) [inline endpoints/openrtb2/auction.go:1127]: Please reword this comment to not include `Digitrust` anymore

**Review by mansinahar — COMMENTED** [review]
LGTM except for the one minor comment

File: endpoints/openrtb2/auction.go (hunk 1 above)
Comment (AlexBVolcy) [reply endpoints/openrtb2/auction.go]: Reworded!

**Review by mansinahar — APPROVED** [review]

**Review by SyntaxNode — DISMISSED** [review]
DMX/DistrictM did not respond to emails regarding this change. Per our policy, we will proceed with this change as there is no material effect on the request send to their adapter.

File: adapters/dmx/dmx_test.go [hunk 2]
```diff
@@ -728,16 +684,14 @@ func TestUsersEids(t *testing.T) {
...
-            },
-            {
+            },{
                 "source": "id5-sync.com",
                 "uids": [{
                     "id": "ID5-12345"
                 }]
-            },
-            {
+            },            {
```
Comment (guscarreon) [inline adapters/dmx/dmx_test.go:694]: Can we roll back to:
```
},
{
```
please?

File: adapters/dmx/dmx_test.go [hunk 3]
```diff
@@ -728,16 +684,14 @@ func TestUsersEids(t *testing.T) {
             },{
                 "source": "pubcid.org",
                 "uids": [{
-                    "id":"11111111"
+                    "id": "11111111"
                 }]
-            },
-            {
+            },{
```
Comment (guscarreon) [inline adapters/dmx/dmx_test.go:689]: Can we roll back to:
```
},
{
```
please?

**Review by guscarreon — COMMENTED** [review]
looks pretty good. I have two very small nitpicks.

File: adapters/dmx/dmx_test.go (hunk 2 above)
Comment (AlexBVolcy) [reply adapters/dmx/dmx_test.go]: Updated!

**Review by mansinahar — APPROVED** [review]

**Review by guscarreon — APPROVED** [review]
LGTM

**Pull request merged**